tv_apps/smart-system/test/marionette/browser_context_menu_test.js

```diff
@@ -0,0 +1,108 @@
+/* global __dirname */
+
+'use strict';
+
+var APP_NAME = 'contextmenuapp';
+var APP_HOST = APP_NAME + '.gaiamobile.org';
+var APP_URL = 'app://' + APP_HOST;
+
+var Keys = {
+  'enter': '\ue006',
+  'right': '\ue014',
+  'esc': '\ue00c'
+};
+
+var assert = require('chai').assert;
+var SmartSystem = require(__dirname + '/lib/system');
```

Comment:
Should we add comment on why we don't use `client.loader.getAppClass('system');`?